frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -957,9 +970,58 @@ pub mod pallet {
 			Ok(())
 		}
 
+		/// Submit a signed solution during the emergency phase.
+		/// It will go through the `feasibility_check` right away.
+		///
+		/// The dispatch origin for this call must be __signed__.
+		///
+		/// The deposit that is reserved might be rewarded or slashed based on
+		/// the outcome.
+		#[pallet::weight(T::DbWeight::get().reads_writes(1, 1))]
+		pub fn submit_emergency_solution(
```

Comment:
This call should be moved to the end of the bottom so that the coll indices don't break.

@niklasad1 this is critical for you and the miner. Once this PR is fixed, you should make a PR and fix all of the the call indices of this pallet with `#[call_index]` #11381